components/message.php

```diff
@@ -72,42 +64,29 @@ class BPCLI_Message extends BPCLI_Component {
 	 *     $ wp bp message create --from=user1 --to=user2 --subject="Message Title" --content="We are ready"
 	 *     Success: Message successfully created.
 	 *
-	 *     $ wp bp message create --from=545 --to=313
+	 *     $ wp bp message create --from=545 --to=313 --subject="Another Message Title" --content="Message OK"
 	 *     Success: Message successfully created.
 	 *
 	 * @alias add
 	 */
 	public function create( $args, $assoc_args ) {
 		$r = wp_parse_args( $assoc_args, array(
 			'to'        => '',
-			'subject'   => '',
-			'content'   => '',
 			'thread-id' => false,
 			'date-sent' => bp_core_current_time(),
-			'silent'    => false,
 		) );
 
-		// Fill in the content.
-		if ( empty( $r['content'] ) ) {
-			$r['content'] = $this->generate_random_text();
-		}
-
-		// Fill in the subject.
-		if ( empty( $r['subject'] ) ) {
-			$r['subject'] = sprintf( 'Message Subject' );
-		}
-
 		$user = $this->get_user_id_from_identifier( $assoc_args['from'] );
 		if ( ! $user ) {
-			WP_CLI::error( 'No user found by that username or ID.' );
+			WP_CLI::error( sprintf( 'No user found by that username or ID %s.', $assoc_args['from'] ) );
```

Comment:
Sorry, small nitpick: `No user found by that username or ID (%s)` - it feels more grammatical if we put it in parentheses, and I think this matches a convention used elsewhere in wp-cli.